### ISSUE TITLE
Master has some tests fails relating to JDK8 issues

### ISSUE THREAD

**Issue by benignbala** [GPars/GPars#37]
The jdk8 branch build is broken. It seems to be because we have removed the extra166y imports for Java8. 

I'll switch from parallalArray to parallelStream() and replace withMapping/withFilter with map/filter

**Comment by benignbala** [comment]
https://github.com/GPars/GPars/pull/38 has a few fixes. But there are a whole bunch of tests still failing

**Comment by benignbala** [comment]
Some of the tests are failing because PAUtils and PAGroovyUtils were removed - I'll continue to try and fix it.

**Comment by russel** [comment]
Everything with PA at the beginning needs to go either by deletion or by being replaced with something Stream realized. PA mean ParallelArray which was a data structure Doug was working with for the extra166 artefact series but which was dropped when Stream came along. For GPars 1.x based on JDK7, we need the extra166 artefact and ParallelArray. For GPars 2.x the point is to remove all the ParallelArray stuff. I have removed the vendor fork we took of extra166 so all the ParallelArray uses need to be removed in order to get a compilation.
